### PR TITLE
[OpenKAT v1.5] Fix boefje detail checkboxes function at scan oois

### PR DESCRIPTION
**Pull request by Rieven**
## Changes
- Fix: Checkbox in table head does not select all OOIS
- Fix: Checkbox `Show objects that don't meet the Boefjes scan level` triggers table view with objects below boefje scan level

## Issue ticket number and link
https://github.com/minvws/nl-rt-tim-abang-rocky/issues/835

### CODE REVIEW

File: katalogus/templates/partials/objects_to_scan.html [hunk 1]
```diff
@@ -4,15 +4,15 @@
...
   <div>
     <h2>{{ form_title }}</h2>
-
-    <p>
+    <p class="warning"
+       role="group"
+       aria-label="{% translate 'scan level warning' %}">
       {% blocktranslate trimmed with scan_level=plugin.scan_level %}
         This boefje will only scan objects with a corresponding clearance level of <strong>L{{ scan_level }}</strong>
         or higher. There is no indemnification for this boefje to scan an OOI with a lower clearance level then <strong>L{{ scan_level }}</strong>. Use the filter to show OOI's with a lower clearance level.
```
Comment (underdarknl) [inline katalogus/templates/partials/objects_to_scan.html:12]: 'than' ipv 'then'

**Review by underdarknl — DISMISSED** [review]
taalfoutje

**Comment by TwistMeister** [comment]
@Rieven please read my comment on the original issue about the "show objects that don't meet the boefjes scan level" checkbox  (https://github.com/minvws/nl-rt-tim-abang-rocky/issues/835).

The warning that was shown isn't really the issue here. The info text beneath the header and the warning after checking the checkbox where carefully considered by @HeleenSG and me and worked in conjunction with each other as intended. What's wrong is the box getting unchecked after the page reloads and the user not being informed about why the list isn't complemented (when there are no additional objects to show). Please discard the changes on file "objects_to_scan.html" since that isn't the issue here and the behaviour and those text were carefully chosen.

File: katalogus/templates/partials/objects_to_scan.html [hunk 2]
```diff
@@ -22,20 +22,6 @@ <h2>{{ form_title }}</h2>
         </form>
       </div>
     </div>
-
-    {% if 'show_all=on' in request.get_full_path %}
```
Comment (underdarknl) [inline katalogus/templates/partials/objects_to_scan.html:26]: This should not be removed.

**Comment by underdarknl** [comment]
The checkbox on 0:22 in the movie is still unchecked after the refresh., It should be on in that state.

File: katalogus/templates/partials/objects_to_scan.html (hunk 2 above)
Comment (Rieven) [reply katalogus/templates/partials/objects_to_scan.html:26]: Fixed in commit: https://github.com/minvws/nl-kat-rocky/pull/111/commits/4258c854ad7c0d2f9c32199056bf43355e96e79f

**Comment by Rieven** [comment]
> @Rieven please read my comment on the original issue about the "show objects that don't meet the boefjes scan level" checkbox ([minvws/nl-rt-tim-abang-rocky#835](https://github.com/minvws/nl-rt-tim-abang-rocky/issues/835)).


Fixed in commit: https://github.com/minvws/nl-kat-rocky/pull/111/commits/4258c854ad7c0d2f9c32199056bf43355e96e79f

File: katalogus/templates/partials/objects_to_scan.html (hunk 2 above)
Comment (Rieven) [reply katalogus/templates/partials/objects_to_scan.html:26]: We dont need to ask this if there is a get request to ?show_all which will show the list of boefjes that dont meet the requirements, moving the warning after if statement is enough

**Review by Darwinkel — APPROVED** [review]
This is perfect!

As an end-user, it is now much clearer what is happening. The user flow to override and set a higher clearance level is especially nice.

Approved!

**Review by underdarknl — APPROVED** [review]

**Pull request merged**